suite-native/message-system/src/components/MessageScreen.tsx

```diff
@@ -0,0 +1,110 @@
+import { Linking } from 'react-native';
+
+import { IconName } from '@suite-common/icons/src';
+import { Box, Button, Pictogram, PictogramVariant, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Message, Variant } from '@suite-common/suite-types';
+
+type MessageScreenProps = {
+    message: Message;
+    defaultTitle?: string;
+    defaultContent?: string;
+    secondaryButtonLabel?: string;
+    onSecondaryButtonPress?: () => void;
+};
+
+const screenStyle = prepareNativeStyle(utils => ({
```

Comment:
Maybe I'm missing something, but why are you not using normal screen and why are we going for absolute positioning?

Reply:
a427534